syntax = 'proto3';



message UserPostRequest{
    string msg = 1;
    int32 write_concern = 2;
}

message UserPostResponse{
    string success = 1;
}

message UserGetRequest{
    bool get = 1;
}

message UserGetResponse{
    repeated string msg = 1;

}


service UserService{
    rpc append(UserPostRequest) returns (UserPostResponse);
    rpc get(UserGetRequest) returns (UserGetResponse);
}
